src/Microsoft.PowerShell.Commands.Management/resources/ProcessResources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="NoProcessFoundForGivenName" xml:space="preserve">
     <value>Cannot find a process with the name "{0}". Verify the process name and call the cmdlet again.</value>
   </data>
+  <data name="RecommendIdTagForGivenName" xml:space="preserve">
```

Comment:
I feel as if we can remove this and reuse `NoProcessFoundForGivenName` and have it look like:

```xml
<data name="NoProcessFoundForGivenName" xml:space="preserve">
    <value>Cannot find a process with the name "{0}". Verify the process name and call the cmdlet again.{1}</value>
</data>
```

Note the `{1}` at the end.

That could conditionally be empty string if not a number or `Try running with -Id to search by Id of process` if a number.

Reply:
we talked about this offline. Since we'd have to have another entry in the resx for the "Try running with -Id to search by Id of process" anyway... the current implementation is ideal.